core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/source/scan/csv/CSVScanSourceOpDesc.scala

```diff
@@ -54,15 +39,17 @@ class CSVScanSourceOpDesc extends SourceOperatorDescriptor {
 
     filePath match {
       case Some(path) =>
-        val headerLine: String =
-          Files.asCharSource(new File(path), Charset.defaultCharset).readFirstLine
-
+        if (headerLine.isEmpty) {
+          headerLine = Option(
```

Comment:
Why is this being encapsulated in Option()

Reply:
it's just an alternative for using null values to represent it being omitted.